datafusion/src/physical_plan/hash_join.rs

```diff
@@ -389,7 +389,7 @@ impl ExecutionPlan for HashJoinExec {
             num_output_rows: 0,
             join_time: 0,
             random_state: self.random_state.clone(),
-            visited_left_side: visited_left_side,
+            visited_left_side,
```

Comment:
this fixes a clippy warning

Reply:
I just noticed the same thing  https://github.com/apache/arrow-datafusion/pull/259